kandinsky/fonts/code_points.h

```diff
@@ -349,7 +349,7 @@ uint32_t ExtendedCodePoints[] = {
   0x2264,  // ≤ // LESS-THAN OR EQUAL TO
   0x2265,  // ≥ // GREATER-THAN OR EQUAL TO
   0xFFFD,  // � // REPLACEMENT CHARACTER
-  0x1d422, // 𝐢 // MATHEMATICAL BOLD SMALL I"
+  0x3b9, // 𝐢 // MATHEMATICAL BOLD SMALL I"
```

Comment:
Why do you change it ?

Reply:
It was a test because on my font the "imaginery" symbol adress didn't reach 0x1d422 so I tried to reaffect it to 0x3b9 which was basically the same symbol but in the greek alphabet. Doesn't work I am trying to insert the character to my font

Reply:
OK

Reply:
Fixed will import the file when the last issue has been solved